src/lib/y2users/users_module/reader.rb

```diff
@@ -0,0 +1,187 @@
+# Copyright (c) [2021] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2users/config"
+require "y2users/user"
+require "y2users/group"
+require "y2users/password"
+require "y2users/login_config"
+require "y2users/useradd_config"
+
+Yast.import "Users"
+Yast.import "Autologin"
+
+module Y2Users
+  module UsersModule
+    # Class for reading the configuration from Yast::Users module
+    class Reader
+      # Generates a new config with the information from YaST::Users module
+      #
+      # @return [Config]
+      def read
+        Config.new.tap do |config|
+          read_elements(config)
+          config.login = read_autologin
+          config.useradd = read_useradd_defaults
+        end
+      end
+
+    private
+
+      # Reads the useradd defaults from Users module
+      #
+      # @return [UseraddConfig]
+      def read_useradd_defaults
+        users_map = Yast::Users.GetLoginDefaults
+        # notes: groups are deprecated and usrskell and create_mail_spool is not in users
```

Comment:
```suggestion
        # notes: groups are deprecated and usrkel and create_mail_spool is not in users
```

NOTE: skel and create_mail_spool are, however, read from the `useradd` configuration file. See https://github.com/yast/yast-users/blob/29a8e9402e76b25313c159c671866e9d61eb7f1a/src/lib/y2users/useradd_config.rb#L154-L179

Reply:
is it usrkel or usrskel?

Reply:
and it is read in useradd_config but not in Users module.

Reply:
> is it usrkel or usrskel?

Second one. Comment updated.